configure.sh

```diff
@@ -280,9 +280,7 @@ while [[ $# -gt 0 ]]; do
     --easy)
         cert_pass="1234"
         cert_cn="LTHNEasyDeploy"
-        if [ -n "$USER" ]; then
-            LTHN_USER="$USER"
-        fi
+        LTHN_USER="`whoami`"
```

Comment:
Is this an intentional change? it looks like it removes the ability to pass the script a $USER as an arg.

Reply:
@pallas1

Reply:
The $USER environment variable should contain the current user name, but it doesn't work on some environments. The best alternative I've found was replacing it with the much more compatible "whoami" call. If that variable is used to pass a username, that should be changed as you are not supposed to change it.

Reply:
OK! Maybe I misunderstood the code. I thought I remembered somewhere along the line seeing a script initialized with `--runas $user`. I reviewed the code again and it looks like there is a `--runas-user` option in configure.sh but these changes *shouldn't* conflict.